README.md

```diff
@@ -496,32 +496,32 @@ The default values for each rule are described below.
 Currently the validator configures Spectral to check the following rules from its
 [“oas" ruleset](https://meta.stoplight.io/docs/spectral/docs/reference/openapi-rules.md):
 ```
+oas2-operation-formData-consume-check: true
+operation-operationId-unique: true
+operation-parameters: true
+operation-tag-defined: true
 no-eval-in-markdown: true
 no-script-tags-in-markdown: true
 openapi-tags: true
 operation-description: true
-operation-operationId-unique: true
-operation-parameters: true
 operation-tags: true
-operation-tag-defined: true
 path-keys-no-trailing-slash: true
 path-not-include-query: true
-request-body-object: true
 typed-enum: true
 oas2-api-host: true
 oas2-api-schemes: true
 oas2-host-trailing-slash: true
-oas2-valid-example: true
-oas2-valid-definition-example: true
+oas2-valid-schema-example: 'warn'
 oas2-anyOf: true
 oas2-oneOf: true
-oas2-operation-formData-consume-check: true
+oas2-unused-definition: true
 oas3-api-servers: true
 oas3-examples-value-or-externalValue: true
 oas3-server-trailing-slash: true
+oas3-valid-media-example: 'warn'
+oas3-valid-schema-example: 'warn'
 oas3-schema: true
-oas3-valid-example: true
-oas3-valid-schema-example: true
+oas3-unused-component: true
```

Comment:
Not sure I see what the new order is here. Any possibility we could alphabetize? Any possibility we could enforce correctness (and alphabetical order) in a build check?

Reply:
These hadn't been updated since the Spectral major release upgrade. I copied and pasted them from our ruleset file so they'd be 100% accurate. I didn't think much about the ordering. I think enforcing some kind of correctness with the build is a good idea but probably outside the scope of this PR

Reply:
@dpopp07 should I open an issue?

Reply:
Sure, that would be great

Reply:
Done: https://github.com/IBM/openapi-validator/issues/368